scripts/build_funcs.sh

```diff
@@ -73,36 +75,46 @@ vet() {
 
 # Builds a container given the dockerfile and image name (not registry).
 # Dockerfiles typically generated via another method.
-build_container_dockerfile_image() {
+build_container_dockerfile_arch() {
 	docker build \
-        -t "$REGISTRY/$2:$IMAGE_VERSION" \
-        -t "$REGISTRY/$2:$IMAGE_TAG" \
-        -t "$REGISTRY/$2:$IMAGE_BRANCH" \
-        -t "$REGISTRY/$2:$GIT_REF_SHORT" \
+        -t "$REGISTRY/$TARGET:$2-$IMAGE_VERSION" \
+        -t "$REGISTRY/$TARGET:$2-$IMAGE_TAG" \
+        -t "$REGISTRY/$TARGET:$2-$IMAGE_BRANCH" \
         -f "$1" \
 		.
 }
 
+buildx_container_windows_version(){
+    BASEIMAGE="$WIN_AMD64_BASEIMAGE:$1"
+    mkdir -p "build/windows/$WIN_ARCH/$VERSION"
+    docker buildx build --pull \
+        --output=type=oci,dest=build/windows/$WIN_ARCH/$VERSION/img.tar \
+        --platform windows/amd64 \
+        -t $REGISTRY/$TARGET:win-$WIN_ARCH-$VERSION-$IMAGE_VERSION \
+        --build-arg BASEIMAGE=$BASEIMAGE \
+        -f DockerfileWindows-$WIN_ARCH \
+        .
+}
+
 # Generates a dockerfile given the os and arch (the 2 and only arguments).
 gen_dockerfile_for_os_arch(){
     if [ "$1" = "linux" ]; then
         if [ "$2" = "amd64" ]; then
             sed -e "s|BASEIMAGE|$AMD_IMAGE|g" \
                 -e 's|CMD1||g' \
-                -e 's|BINARY|build/linux/amd64/sonobuoy|g' Dockerfile > "Dockerfile-$arch"
+                -e 's|BINARY|build/linux/amd64/sonobuoy|g' Dockerfile > "Dockerfile-$2"
         elif [ "$2" = "arm64" ]; then
             sed -e "s|BASEIMAGE|$ARM_IMAGE|g" \
                 -e 's|CMD1||g' \
-                -e 's|BINARY|build/linux/arm64/sonobuoy|g' Dockerfile > "Dockerfile-$arch"
+                -e 's|BINARY|build/linux/arm64/sonobuoy|g' Dockerfile > "Dockerfile-$2"
         else
             echo "Linux ARCH unknown"
         fi
-    elif [ "$2" = "windows" ]; then
-        if [ "$arch" = "amd64" ]; then
-			sed -e "s|BASEIMAGE|$WIN_IMAGE|g" \
-			    -e 's|BINARY|build/windows/amd64/sonobuoy.exe|g' DockerfileWindows > "DockerfileWindows-$arch"
-			build_container_dockerfile_image "DockerfileWindows-$arch" sonobuoy
-			build_container_dockerfile_image "DockerfileWindows-$arch" "sonobuoy-win-$arch"
+    elif [ "$1" = "windows" ]; then
+        if [ "$2" = "amd64" ]; then
+            # Onlhy doing one arch so this could be hardcoded, likewise we could handle the
+            # base image differently. Wanted something here for parity with linux in case we expand it though.
+			sed -e 's|BINARY|build/windows/amd64/sonobuoy.exe|g' DockerfileWindows > "DockerfileWindows-$2"
 		else 
 			echo "Windows ARCH unknown"
         fi
```

Comment:
indentation seems off?

Reply:
So locally it looks good but I found that there is a mix of tabs/spaces in the file. Made the file consistent so the UI here should look better on the next push.